README.md

```diff
@@ -14,72 +15,118 @@ OSPSuite.ReportingEngine implementation in R
 
-[OPTIONAL] Install **rsvg-convert** (required by Pandoc for conversion of images in SVG format)
+- The package bundle for the __development__ version is available [here &#128230;](https://ci.appveyor.com/project/open-systems-pharmacology-ci/OSPSuite-ReportingEngine/branch/develop/artifacts).
+- The package bundle for the __release__ version is available [here &#128230;](https://ci.appveyor.com/project/open-systems-pharmacology-ci/OSPSuite-ReportingEngine/branch/master/artifacts).
 
-* [Installer (Windows)](https://github.com/miyako/console-rsvg-convert/releases)
-  * The installation folder must be added to the system path.
-* For Linux, **librsvg** package must be installed (package name depends on distribution, e.g. **librsvg2-bin** for Ubuntu).
+Then, to install manually, replace the `path/to/ospsuite.reportingengine.zip` by your actual local path to the `.zip` or `tar.gz` file in the code below:
 
-# Development tasks
+```r
+install.packages(path/to/ospsuite.reportingengine.zip, repos = NULL)
+```
 
-## dev_mode
+### Required packages
 
-  `devtools::dev_mode` function switches your version of R into "development mode". This is useful to avoid clobbering the existing versions of CRAN packages that you need for other tasks. Calling dev_mode() again will turn development mode off, and return you to your default library setup.
+`{ospsuite.reportingengine}` requires following packages to be installed:
 
-```R
-# This will install the package in the folder C:/Rpackages
-devtools::dev_mode(path="C:/Rpackages")
-```
+- From the [Open Systems Pharmacology](https://github.com/Open-Systems-Pharmacology) ecosystem:
 
-## Reload the package
+|Package|Version|Installation Instructions|
+|-------|-------|------------|
+|[`{ospsuite.utils}`](https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils)|$\geq$ 1.4|Download and install package bundle [here &#128230;](https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/releases)|
+|[`{tlf}`](https://github.com/Open-Systems-Pharmacology/TLF-Library)|$\geq$ 1.5|Download and install package bundle [here &#128230;](https://github.com/Open-Systems-Pharmacology/TLF-Library/releases)<br>&#9888; Visit [`{tlf}` Documentation](https://github.com/Open-Systems-Pharmacology/TLF-Library) to install its dependencies &#9888;|
+|[`{rSharp}`](https://github.com/Open-Systems-Pharmacology/rSharp)|$\geq$ 1.0|Instructions are available [here &#128214;](https://github.com/Open-Systems-Pharmacology/rSharp#installation)|
+|[`{ospsuite}`](https://github.com/Open-Systems-Pharmacology/OSPSuite-R)|$\geq$ 12.0|Download and install package bundle [here &#128230;](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/releases)<br>Instructions are available [here &#128214;](https://github.com/Open-Systems-Pharmacology/OSPSuite-R#installation)<br>&#9888; Visit [`{ospsuite}` Documentation](https://github.com/Open-Systems-Pharmacology/OSPSuite-R) to install its dependencies &#9888;|

```

Comment:
let's change to >=12.1
12.0 was not even released; we switched directly to 12.1 with the migration to rSharp